### ISSUE TITLE
Cloud Flare Clearance Failed

### ISSUE THREAD

**Issue by mubbsy** [Stickymaddness/Procurement#1002]
Haven't been able to get passed this error since sunday afternoon (2 days ago). Have tried waiting, refreshing POESESSID and putting in the new one, and I've probably tried 100+ times since it started failing. Is this just me or is everyone having issues bypassing the cloud flare protection at the moment??

**Comment by thailyn** [comment]
Hi, thanks for the report.  There are some known issues with the version of the library that handles the CloudFlare requirements.  Basically, there are types of authentication CloudFlare sometimes asks for that the library doesn't cover.  I have a (messy) PR that uses a patched version of the library to handle these cases.  I also noticed the library has been updated with this change, so I can update my PR tonight to use that instead (and not be as messy).
